Reed/DefinitionView/views/DefinitionView.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  DefinitionModal.swift
+//  Reed
+//
+//  Created by Roger Luo on 9/9/20.
+//  Copyright © 2020 Roger Luo. All rights reserved.
+//
+
+import SwiftUI
+
+struct DefinitionView: View {
+    @ObservedObject var viewModel: DefinitionViewModel
+    @State private var scrollViewID = UUID()
+
+    var body: some View {
+        ZStack {
+            Color.white
+            ScrollView {
+                HStack(alignment: .lastTextBaseline) {
+                    Text(viewModel.title)
+                        .font(.title2)
+                        .padding(.horizontal)
+                    Text("\(viewModel.currReading)")
+                        .font(.footnote)
+                        .foregroundColor(.gray)
+                        .padding(.trailing)
+                    Spacer()
+                }.padding(.vertical)
+                
+                ForEach(0..<viewModel.defsArr.count, id: \.self) { i in
+                    HStack(alignment: .lastTextBaseline) {
+                        Text("\(i + 1). \(viewModel.defsArr[i].def)")
```

Comment:
Since numbers are different widths, creating the list in this way causes misalignment at the start of the text of each list.
The solution to this is something like this:
1. Make a VStack for the numbers and align them to the right.
2. Make a VStack for the text and align them to the left.
3. Throw the two VStacks into an HStack, and have the HStack enforce some spacing between the two elements.
This way, even if we have large numbers like 13 or 20, all the digits end at the same place, and all the definitions start at the same place.

Reply:
some definitions may exceed one line, so it might be difficult to space the numbers out correctly.

Reply:
Hmm. There's probably a way to make this work by using Stacks... but if we keep getting stuck, we can try using Grids, which are new to iOS 14. https://www.hackingwithswift.com/quick-start/swiftui/how-to-position-views-in-a-grid-using-lazyvgrid-and-lazyhgrid

Reply:
I think just separating the number and the definitions into two text views will do the job

Reply:
If this is done, can you update the screenshots?

Reply:
actually, I can just do some math and keep them in the same textview

Reply:
<img width="50%" alt="Screen Shot 2020-11-22 at 1 55 52 PM" src="https://user-images.githubusercontent.com/42554019/99918255-7b627c80-2cca-11eb-9d25-060e049d14bc.png">

Reply:
Beautiful.